rosa_planning/rosa-cloud-expert-prereq-checklist.adoc

```diff
@@ -59,6 +59,7 @@ Ensure that you have the following accounts, credentials, and permissions.
 //OSDOCS-11789: Moving these here because it is a permission / account level enablement
 * Enable ROSA for your AWS account on the link:https://console.aws.amazon.com/rosa/[AWS console].
 ** If your account is the management account for your organization (used for AWS billing purposes), you must have `aws-marketplace:Subscribe` permissions available on your account. See _Service control policy (SCP) prerequisites_ for more information, or see the AWS documentation for troubleshooting: link:https://docs.aws.amazon.com/rosa/latest/userguide/security-iam-troubleshoot.html#error-aws-orgs-scp-denies-permissions[AWS Organizations service control policy denies required AWS Marketplace permissions].
+* Ensure you have tag policies enabled. For more information, see link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_tag-policies.html[Tag policies] in the AWS documentation.
```

Comment:
The opposite: we can't have customers enable restrictive tag policies. 

Example: our openshift installer will want to tag the ec2 instances it creates. If the customer restricts this, and adds a policy that says "hey, if you create an ec2 instance, you need to add the tag `billing:OrganizationA`", the instance creation fails as the installer doesn't know about this requirement.